tests/fixed/not_spacing.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+$test = 1;
+
+if (!$test > 0) {
+    echo 1;
+} elseif (!$test === 0) {
+    echo 0;
+} else {
+    echo - 1;
+}
+
+while (!    true) {
```

Comment:
extra spaces here

Reply:
That means we don't have a rule for that :)